apps/python/descriptions/scart.xml

```diff
@@ -55,9 +55,9 @@
 			<group name="Output">
 				<option flag="c" argument="channels">
 					<description>
-					Specify the channel filter for the dumped streams as regular
-					expression. Default: (B|S|M|H)(L|H)(Z|N|E). To dump only
-					BHZ, BHN and BHE streams use BH(Z|N|E).
+					Channel filter to be applied to the streams selected by -n option.
```

Comment:
The text suggests that -c only applies with -n. I propose to drop "selected by -n option".

Reply:
Looking at the code it actually affects only the streams selected by the -n option. It doesn't have any other purpose. Am I missing something here?